pkg/model/deployment.proto

```diff
@@ -146,3 +147,15 @@ message Commit {
     string url = 6;
     int64 created_at = 14 [(validate.rules).int64.gt = 0];
 }
+
+message ArtifactVersion {
+    enum Kind {
+        UNKNOWN = 0;
+        CONTAINER_IMAGE = 1;
+    }
+   
+    Kind kind = 1 [(validate.rules).enum.defined_only = true];
+    string version = 2 [(validate.rules).string.min_len = 1];
+    string name = 3;
+    string url = 4;
+}
```

Comment:
if we use it in both application and deployment proto, should we move it to common.proto?

Reply:
sounds nice! I'll fix it